docker-compose.yml

```diff
@@ -53,6 +53,7 @@ services:
       - RPC_PASS=sparkswap
     ports:
       - '27492:27492'
+      - '8081:8080'
```

Comment:
[nit] since `8081` is a pretty popular port for local dev, we could instead change this to `27592` or the like and document what default addresses we use for this http server